controller/handler/compliance-count.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * @module controller/handler/trial
+ */
+
+const database = require('../../model');
+const processPatient = require('../helper/process-patient');
+const processTrial = require('../helper/process-trial');
+const moment = require('moment');
+const getCount = require('../helper/process-compliance-count');
+
+/**
+ * A dashboard with an overview of a specific trial.
+ * @function getComplianceCount
+ * @param {row} currentTrial - Row
+ * @param {row} stages - Row
+ * @param {row} patients - Row
+ * @param {View} reply - Row
+ * @returns {View} Rendered page
+ */
+function getComplianceCount (currentTrial, stages, patients, reply) {
+    const startDate = moment().startOf('Week');
+
+    Promise
+        .all([
```

Comment:
`Promise.all` is used when multiple Promises need to be resolved at the same time.
This is a single promise so `database.sequelize.query(...).then(...)` would be preferred.